ui-v2/src/api/zod/events/events.ts

```diff
@@ -35,7 +35,7 @@ export const createEventsEventsPostBodyItem = zod
 			.optional()
 			.describe("A list of additional Resources involved in this event"),
 		payload: zod
-			.object({})
+			.record(zod.string(), zod.any())
```

Comment:
```suggestion
			.record(zod.unknown())
```
Might be better, because it forces the developer to check if the field is a certain type first.
https://dev.to/sachinchaurasiya/type-safety-in-typescript-unknown-vs-any-55c0

Reply:
this change was auto-generated by the pre-commit, which I suppose we might have to update?